src/main/resources/static/css/profile.less

```diff
@@ -46,6 +48,19 @@
   &-6 {animation-delay: 0.5s;}
 }
 
+.chart {
+  &-container{
+    width: 95%;
+    margin-left: 2.5%;
+  }
+  &-headline {
+    text-align: center;
+  }
+  &-canvas {
+
```

Comment:
Is there a reason for this to be there although it's empty?

Reply:
Thats my fault

Reply:
You can delete it